x-pack/plugins/infra/public/observability_logs/log_stream_query_state/src/url_state_storage_service.ts

```diff
@@ -187,5 +285,32 @@ const decodeQueryValueFromUrl = (queryValueFromUrl: unknown) =>
     () => filterStateInUrlRT.decode(queryValueFromUrl)
   );
 
-export const replaceLogFilterInQueryString = (query: Query) =>
-  replaceStateKeyInQueryString<Query>(defaultFilterStateKey, query);
+const decodePositionQueryValueFromUrl = (queryValueFromUrl: unknown) => {
+  return legacyPositionStateInUrlRT.decode(queryValueFromUrl);
+};
+
+const ONE_HOUR = 3600000;
+export const replaceLogFilterInQueryString = (query: Query, time?: number) =>
+  replaceStateKeyInQueryString<FilterStateInUrl>(defaultFilterStateKey, {
+    query,
+    ...(time && !Number.isNaN(time)
+      ? {
+          timeRange: {
+            from: new Date(time - ONE_HOUR).toISOString(),
+            to: new Date(time + ONE_HOUR).toISOString(),
+          },
+        }
+      : {}),
+    refreshInterval: {
+      pause: true,
+      value: 5000,
+    },
```

Comment:
**nit**: should this be `DEFAULT_REFRESH_INTERVAL`?

Reply:
Oh, good catch 👍